plugins/circleci/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -35,14 +29,12 @@ import {
   Link,
 } from '@backstage/core-components';
 
-const IconLink = IconButton as any as typeof MaterialLink;
```

Comment:
based on this, it almost looks like this plugin was intentionally mapping `IconButton` to the straight forward `@material-ui/core/Link` component, but just CALLING it an `IconButton`?  given that, i wonder if rather than return this refactor back to point to the TRUE `IconButton` component as you've done in 20, maybe the better solution is to just go straight to the `@backstage/core-components` `Link`?

Do you have CircleCI and were able to see how this changes the build steps page?

If not, I don't have my env setup to test this at the moment but I'll see if I can give your PR a shot and see what it looks like on my end.

Reply:
I think there was an issue with types at some point.

> maybe the better solution is to just go straight to the @backstage/core-components Link?

I agree. `@backstage/core-components/Link` tracks `click` events, it would be great to use it here.
(see https://github.com/backstage/backstage/pull/14050#discussion_r999898982)